earth_enterprise/src/SConstruct

```diff
@@ -439,8 +441,8 @@ env = khEnvironment(
     jobstats=jobstats,
     internal=internal,
     native_cc=native_cc,
-    cc_dir=cc_dir,
     bundle_3rd_libs=bundle_3rd_libs,
+    cc_dir=cc_dir,
```

Comment:
was this switch of order (in 'cc-dir') needed for some reason ?

Reply:
No, it's just a result of automerging.  I didn't think it's important enough to make another commit.